easybuild/framework/easyconfig/default.py

```diff
@@ -165,6 +165,7 @@
 
     # OTHER easyconfig parameters
     'buildstats': [None, "A list of dicts with build statistics", OTHER],
+    'options' : [{},"A list of extra options for specific blocks", OTHER],
```

Comment:
can you be a bit more precise in the help? (or even better, make a PR to the documentation and explain what it supposed to do) 
the option name `options` is very generic. if this is some 'pass anything to any block` sort of option, we should be very careful with that.